### ISSUE TITLE
support cluster benchmark?

### ISSUE THREAD

**Issue by rfyiamcool** [albus01/iBenchmark#1]
Please see title ,  thank U.

**Comment by albus01** [comment]
@rfyiamcool No, it does not support cluster. But you can use bash shell to run it parallelly in several client nodes.
